make/migrations/postgresql/0010_1.9.0_schema.up.sql

```diff
@@ -9,6 +9,46 @@ CREATE TABLE cve_whitelist (
     UNIQUE (project_id)
 );
 
+CREATE TABLE Blob (
```

Comment:
Table name using snake case will be better and keep the same with other table names.